src/components/ens-registration/RegistrationReviewRows/RegistrationReviewRows.tsx

```diff
@@ -110,7 +110,7 @@ export default function RegistrationReviewRows({
 
   return (
     <Box>
-      <Stack space="34px">
+      <Stack space="30px">
```

Comment:
looking good but wondering if this is breaking the design spec?

Reply:
i think this is actually closer to the design spec, the gap should be 30px not 34px

Reply:
👌  you're right